.github/workflows/backend-api-workflow.yml

```diff
@@ -0,0 +1,15 @@
+name: Build and Publish Docker
+on:
+  pull_request:
+    paths:
+    - 'backend-api/**'
```

Comment:
This will make it run only when there are files changed in backend-api. 

We can use this approach for the other projects too.

Reply:
oh nice :+1: